src/pages/communication-page/communication.ts

```diff
@@ -113,6 +115,9 @@ export class CommunicationPage extends PracticeableBasePageComponent {
       }
     }
 
+    // Will be required to cater for more than Cat B at a later stage
+    this.store$.dispatch(new PopulateTestCategory(TestCategory.B));
```

Comment:
Can't this action be dispatched as part of the start test actions?

Reply:
The category could also be retrieved from the slot data in the store and that would remove the hard coding

Reply:
Nice, I'll do that :)